src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,37 @@
+package study;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class StringTest {
+
+    @Test
+    void split(){
+        String[] values = "1,2".split(",");
+        assertThat(values).hasSize(2);
+        assertThat(values).containsExactly("1","2");
+
+        values = "1".split(",");
+        assertThat(values).hasSize(1);
+        assertThat(values).containsExactly("1");
+    }
+
+    @Test
+    void subString() {
+        String value = "(1,2)";
+        assertThat(value.substring(1,value.length()-1)).contains("1,2");
```

Comment:
```suggestion
        assertThat(value.substring(1,value.length() - 1)).isEqualTo("1,2");
```
contains는 특정 문자열이 포함되었는지 확인하는 메서드입니다.
즉, 검증하려는 값이 `"1,2,3,4,5"` 이어도 테스트가 통과하게 됩니다. 🥲

문자열이 완벽하게 같은지 검증하기 위해서는 isEqualTo()가 올바른 검증 방법입니다.